CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.1]
```

Comment:
It would be nice to have a minor version (0.2.0) update since this PR is adding a new feature to the app and when everything will be ok to merge, to select  'Minor (backwards-compatible functionality)'